titan-v0-scm-1.rockspec

```diff
@@ -0,0 +1,32 @@
+package = "titan-v0"
+version = "scm-1"
+source = {
+   url = "git+https://github.com/titan-lang/titan-v0"
+}
+description = {
+   summary = "Initial prototype of the Titan compiler",
+   detailed = [[
+      Initial prototype of the Titan compiler.
+      This is a proof-of-concept, implementing a subset of
+      the Titan language.
+   ]],
+   homepage = "http://github.com/titan-lang/titan-v0",
+   license = "MIT"
+}
+dependencies = {
```

Comment:
We are also using argparse and inspect 
https://luarocks.org/modules/luarocks/argparse
https://luarocks.org/modules/kikito/inspect